saleor/account/emails.py

```diff
@@ -23,12 +27,22 @@ def send_password_reset_email(context, recipient):
 
 @shared_task
 def send_account_delete_confirmation_email(token, recipient_email):
-    site = Site.objects.get_current()
     delete_url = build_absolute_uri(
         reverse('account:delete-confirm', kwargs={'token': token}))
-    ctx = {'site_name': site.name, 'domain': site.domain, 'url': delete_url}
+    ctx = get_email_base_context()
+    ctx.update({'url': delete_url})
     send_templated_mail(
         template_name='account/account_delete',
         from_email=settings.DEFAULT_FROM_EMAIL,
         recipient_list=[recipient_email],
         context=ctx)
+
+
+def get_email_base_context():
```

Comment:
It would be better to define it to `core.email` instead of duplicating it, otherwise it doesn't fix the issue I mentioned. :+1: